### PR TITLE
feature/33-Add components for projects list.

### PR DESCRIPTION
**Pull request by ela-ocana-jalafund**
[AB#33](https://dev.azure.com/funjaladev/7867a1c5-5ac7-4c7f-baff-87d740bf4fc1/_workitems/edit/33): Frontend Projects List.
[AB#33](https://dev.azure.com/funjaladev/7867a1c5-5ac7-4c7f-baff-87d740bf4fc1/_workitems/edit/33): It was tested with JasonPlaceholder

### CODE REVIEW

**Comment by martin-callejas-jalafund** [comment]
please update with latest develop

we want to test the PR trigger and your branch is missing that file

File: src/components/Project/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+//import { Email, CalendarToday, Description } from "@material-ui/icons";
+import "./Card.scss";
+
+function Card(props) {
+  const { id, name } = props;
+  //const { id, name, description, contact, creationDate } = props;
```
Comment (martin-callejas-jalafund) [inline src/components/Project/Card.jsx:6]: please remove this comment

File: src/components/Project/ProjectsAPI.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+const instance = axios.create({
+  baseURL: "https://jsonplaceholder.typicode.com",
+});
+
+export default async function GetProjects() {
+  try {
+    let result = await (await instance.get("/users")).data;
+    return result;
+  } catch (error) {
+    console.log(error);
```
Comment (jorge-lema-jalafund) [inline src/components/Project/ProjectsAPI.js:12]: remove this console.log please
Handle the error and do or return something to let your component know that the API call failed. You could also handle the case when you get an empty list of projects.

**Review by jorge-lema-jalafund — CHANGES_REQUESTED** [review]
Solve the comment

**Comment by martin-callejas-jalafund** [comment]
please cancel this PR and rename your branch, is not following our naming conventions... it reads: `Feuture/`
should be `feature/`

**Comment by ela-ocana-jalafund** [comment]
Rename branch.

**Pull request closed without merging**